### PR TITLE
Added recognition licenses for sky cultures

### PR DESCRIPTION
**Pull request by alex-w**
### Description
Added code to recognition the license (or licenses) info in sky cultures and show human readable hints for granted permissions for users. A few sky cultures have an updated license info in `info.ini` file. 

Sub-tasks:
- [x] Add information about requirement data for `info.ini` file into Stellarium User Guide
- [x] Add license info into info.ini file for all sky cultures (all known)
- [x] Removing the license info from descriptions (see `description.en.utf8` files) of sky cultures

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (header file)
- [x] I have updated the respective chapter in the Stellarium User Guide
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gzotti — CHANGES_REQUESTED** [review]
These entries make sense esp. when a check is done before packing. Useful test:

- check for existence of this entry
- check for particular license keywords (check validity of this entry)

Formally, git does not like the missing newline. This *must* be fixed before merge.

**Comment by alex-w** [comment]
> These entries make sense esp. when a check is done before packing. Useful test:
> 
> * check for existence of this entry
> * check for particular license keywords (check validity of this entry)

The manager of sky cultures is the next obvious task :) Probably also with adding the script for manage packaging sky cultures into main package and separate packages for each "non-free" sky culture.

> Formally, git does not like the missing newline. This _must_ be fixed before merge.

Fixed!

**Comment by alex-w** [comment]
@gzotti please review this pull request

File: skycultures/anutan/info.ini [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [info]
 name = Anutan
 classification = ethnographic
+license = CC BY-SA 4.0 International Public Licence
```
Comment (gzotti) [inline skycultures/anutan/info.ini:4]: why do we name the option "license" (American standard programmer language) and then often, but not everywhere write "licence" (the "very British" format) while the CC licenses are commonly written again in American English, "license" ?

Comment (alex-w) [reply skycultures/anutan/info.ini:4]: "Licence" this is typo

Comment (alex-w) [reply skycultures/anutan/info.ini]: OK, done!

File: skycultures/maya/info.ini [hunk 2]
```diff
@@ -2,4 +2,4 @@
 name = Maya
 author = Rodas-Mejuto
 boundaries = none
-classification = historical
\ No newline at end of file
+classification = historical
```
Comment (gzotti) [inline skycultures/maya/info.ini:5]: I should ask Mejuto for terms.

**Comment by alex-w** [comment]
@sushoff please review this pull request

**Comment by sushoff** [comment]
yes, I agree - thank you for this PR

**Review by gzotti — APPROVED** [review]
Now we are complete.

**Pull request merged**